sycl/test-e2e/Basic/vector/byte.cpp

```diff
@@ -1,9 +1,6 @@
 // RUN: %{build} -std=c++17 -o %t.out
 // RUN: %{run} %t.out
 
-// RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes -std=c++17 -o %t2.out %}
-// RUN: %if preview-breaking-changes-supported %{ %{run} %t2.out %}
-
```

Comment:
Why are we dropping this test?

Reply:
`Basic/vector/byte.cpp` -> `Basic/vector/byte_preview.cpp`. The old test does operations on `vec<byte, _>` which are not supported by `std::byte` itself. In the new implementation, `vec<byte>` only supports math operations that are supported by `std::byte` itself (listed here: https://en.cppreference.com/w/cpp/types/byte)

Reply:
I just realized what you did here. Can we merge this and `byte_preview.cpp` into a single file with an `ifdef` guard instead of having two different files?

Reply:
I tried merging `byte.cpp` and `byte_preview.cpp` but that was resulting into a lot of `#ifdefs` - and making the test logic unnecessarily complex. I'd prefer having a separate `byte_preview.cpp` as it would be easier to make it default in the next ABI-breaking window.